source/TradingPanel.cpp

```diff
@@ -73,8 +73,7 @@ TradingPanel::~TradingPanel()
 {
 	if(profit)
 	{
-		string message = "You sold " + to_string(tonsSold)
-			+ (tonsSold == 1 ? " ton" : " tons") + " of cargo ";
+		string message = "You sold " + Format::CargoString(tonsSold, "cargo");
```

Comment:
The existing version includes a space after `cargo`, yours doesn't.
```suggestion
		string message = "You sold " + Format::CargoString(tonsSold, "cargo ");
```